todo-app/src/Components/todoList.js

```diff
@@ -1,74 +1,75 @@
-//jshint esnext: true
+import React, { Component } from "react";
 
-import React, { Component, Fragment } from 'react'
-
-import TodoItem from './todoItem';
-import './styles/todoList.css';
+import TodoItem from "./TodoItem";
```

Comment:
Same here. You just need to capitalise your `todoItem.js` file.

Reply:
Hmm that's weird. Cause I got them them both capitalised (also `TodoList.js`) and I pushed them this way, I'm checking `git status` and nothing is different 
![See this](https://user-images.githubusercontent.com/29838474/34880458-cea91234-f7b0-11e7-84e1-85275d708bbc.png)